### DISCUSSION TITLE
Where to find Linux Mint Mate 32 bit?

### DISCUSSION THREAD

**Discussion by Moigaston** [linuxmint/discussions#529] [Questions]
Hi,
Who can send me a link, where to find Linux Mint Mate 32-bit? I didn't find the 32-bit version on https://linuxmint.com/download_all.php. 
Thanks

**Comment by billyswong** [comment]
No, there isn't one for you in 2024. 32-bit Linux Mint is only available in the form of LMDE, Linux Mint Debian Edition. The mainstream Linux Mint releases are based on Ubuntu and Ubuntu has discontinued 32-bit x86 platform support.

So your only choice is to either install LMDE then replace the DE by yourself manually, or consider other Linux distributions.

**Comment by Moigaston** [comment]
I've got it! :)
https://www.linuxmint.com/edition.php?id=308
